.github/workflows/publish.yml

```diff
@@ -0,0 +1,36 @@
+﻿name: Publish
+
+on:
+  release:
+    types: [created]
+    
+env:
+  PROJECT_PATH: src/Crowdin.Api
+  PROJECT_CSPROJ_PATH: ${{ env.PROJECT_PATH }}/Crowdin.Api.csproj
+  
+jobs:
+  publish-nuget:
+    runs-on: ubuntu-latest
+    permissions:
```

Comment:
we do not need these permissions here since we do not publish any GitHub packages

Reply:
fixed